activemq-partition/src/main/java/org/apache/activemq/partition/ZKClient.java

```diff
@@ -39,30 +25,32 @@
 import org.linkedin.util.clock.Timespan;
 import org.linkedin.util.concurrent.ConcurrentUtils;
 import org.linkedin.util.io.PathUtils;
-import org.linkedin.zookeeper.client.ChrootedZKClient;
-import org.linkedin.zookeeper.client.IZooKeeper;
-import org.linkedin.zookeeper.client.IZooKeeperFactory;
-import org.linkedin.zookeeper.client.LifecycleListener;
-import org.linkedin.zookeeper.client.ZooKeeperFactory;
-import org.osgi.framework.InvalidSyntaxException;
```

Comment:
why are we removing osgi support?

Reply:
ignore this, just realised its exceptions that you've removed from method throws